oeo.omn

```diff
@@ -2071,7 +2071,7 @@ Source: https://en.wikipedia.org/wiki/Electrical_grid"@en,
         rdfs:comment "Synonym: Network"@en
     
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000031>,
+        Grid,
         has_part some ElectricityGridComponent,
         has_part min 1 EnergyGeneratorTechnology,
```

Comment:
```suggestion
```
after restructuring 'Generator', we can add `has_part min 1 Generator` or however it will look like then.